### PR TITLE
Add requests dependency

### PR DESCRIPTION
**Pull request by jenhaoyang**
Add requests dependency

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
     author_email='HarlemSquirrel@gmail.com',
     url='https://github.com/HarlemSquirrel/python-vivotek',
     license='LGPLv3+',
+    install_requires=[
+        'requests',
```
Comment (HarlemSquirrel) [inline setup.py:19]:
```suggestion
        'requests >= 2.21',
```

**Review by HarlemSquirrel — CHANGES_REQUESTED** [review]
Thank you for opening this PR! 

I would like to set a minimum version here since I haven't tested this with any earlier versions of requests.

File: setup.py (hunk 1 above)
Comment (jenhaoyang) [reply setup.py:19]: No problem

**Review by HarlemSquirrel — APPROVED** [review]
Thank you!

**Pull request merged**